pypeit/core/wavecal/defs.py

```diff
@@ -65,6 +65,8 @@ def lines():
     line_dict['ThAr_MagE'] = 2**18  # R=4100
     line_dict['OH_FIRE_Echelle'] = 2**19  # R=6000
     line_dict['Ar_IR_GNIRS'] = 2**20  # R=6000
+    line_dict['FeI'] = 2**21
```

Comment:
This will collide with the refactoring in `wvcalib_refactor_final`. 
That's ok, just need to keep our eye on it.